docs/dietpi_tools.md

```diff
@@ -1,3 +1,8 @@
+---
+title: DietPi Tools
```

Comment:
The title is not required here, since it matches the navigation menu entry.

Reply:
Ok. 
But as I understood, it is no real impediment.

Reply:
At least it increases efforts when we decide to change the pages name. Also it removes the generic appendix " - DietPi.com Docs". I'd prefer to only add a title explicitly when the page has no meaningful navigation menu text.

Reply:
Ah wait, no the appendix is not missing with this, just checked this back. However, here it is really just a duplicate of the menu text.

Reply:
Ok. Will delete these duplicate lines also in other files.

Reply:
As of other comment, in `usage.md` only, I suggest, on software category pages it does add reasonable context, so let's keep them 👍.